Tests/BootCamp.Chapter.Tests/ShopTests.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests
+{
+    // You don't have to be here for a long time.
+    public class ShopTests
+    {
+        private const string ItemName = "Shortsword";
+        private const decimal ItemPrice = 10;
+        private const decimal InitialShopMoney = 10;
+
+        private Shop _shop;
```

Comment:
Codacy found an issue: [Make '_shop' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5776099)